contrib/kind.yaml.j2

```diff
@@ -17,10 +15,15 @@ networking:
   ipFamily: {{ ovn_ip_family }}
 {%- endif %}
 featureGates:
-  SCTPSupport: true
```

Comment:
@astoycos is this intended?

Reply:
Yeah in k8s 1.19 it was on by default so I don't think we need this anymore https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/

Reply:
you should rebase this PR? https://github.com/ovn-org/ovn-kubernetes/blob/master/contrib/kind.yaml.j2

Reply:
Yep, Just Figuring some last things out